### ISSUE TITLE
Headless.new causing warning: unsupported spawn option: err

### ISSUE THREAD

**Issue by adamdougal** [leonid-shevtsov/headless#57]
I'm having an issue using version 2.1.0 when running:
````ruby
Headless.new
````
Gives me this error:
````
file:/var/lib/jenkins/.rvm/rubies/jruby-1.7.9/lib/jruby.jar!/jruby/kernel19/process.rb:14 warning: unsupported spawn option: err
Initializing built-in extension Generic Event Extension
Initializing built-in extension SHAPE
Initializing built-in extension MIT-SHM
Initializing built-in extension XInputExtension
Initializing built-in extension XTEST
Initializing built-in extension BIG-REQUESTS
Initializing built-in extension SYNC
Initializing built-in extension XKEYBOARD
Initializing built-in extension XC-MISC
Initializing built-in extension SECURITY
Initializing built-in extension XINERAMA
Initializing built-in extension XFIXES
Initializing built-in extension RENDER
Initializing built-in extension RANDR
Initializing built-in extension COMPOSITE
Initializing built-in extension DAMAGE
Initializing built-in extension MIT-SCREEN-SAVER
Initializing built-in extension DOUBLE-BUFFER
Initializing built-in extension RECORD
Initializing built-in extension DPMS
Initializing built-in extension X-Resource
Initializing built-in extension XVideo
Initializing built-in extension XVideo-MotionCompensation
Initializing built-in extension SELinux
Initializing built-in extension GLX
End of file reached (EOFError)
org/jruby/RubyIO.java:2856:in `read_nonblock'
/var/lib/jenkins/.rvm/gems/jruby-1.7.9@cirrus-new/gems/headless-2.1.0/lib/headless.rb:183:in `ensure_xvfb_is_running'
/var/lib/jenkins/.rvm/gems/jruby-1.7.9@cirrus-new/gems/headless-2.1.0/lib/headless.rb:174:in `launch_xvfb'
/var/lib/jenkins/.rvm/gems/jruby-1.7.9@cirrus-new/gems/headless-2.1.0/lib/headless.rb:159:in `pick_available_display'
org/jruby/RubyRange.java:479:in `each'
/var/lib/jenkins/.rvm/gems/jruby-1.7.9@cirrus-new/gems/activesupport-4.1.10/lib/active_support/core_ext/range/each.rb:7:in `each_with_time_with_zone'
/var/lib/jenkins/.rvm/gems/jruby-1.7.9@cirrus-new/gems/headless-2.1.0/lib/headless.rb:155:in `pick_available_display'
/var/lib/jenkins/.rvm/gems/jruby-1.7.9@cirrus-new/gems/headless-2.1.0/lib/headless.rb:151:in `attach_xvfb'
/var/lib/jenkins/.rvm/gems/jruby-1.7.9@cirrus-new/gems/headless-2.1.0/lib/headless.rb:88:in `initialize'
/var/lib/jenkins/jobs/_umv/jobs/_tests/jobs/umv-admin-functional-tests/workspace/admin_tests/features/support/env.rb:113:in `(root)'
org/jruby/RubyKernel.java:1099:in `load'
/var/lib/jenkins/.rvm/gems/jruby-1.7.9@cirrus-new/gems/cucumber-1.2.0/bin/../lib/cucumber/rb_support/rb_language.rb:1:in `(root)'
/var/lib/jenkins/.rvm/gems/jruby-1.7.9@cirrus-new/gems/cucumber-1.2.0/bin/../lib/cucumber/rb_support/rb_language.rb:129:in `load_code_file'
/var/lib/jenkins/.rvm/gems/jruby-1.7.9@cirrus-new/gems/cucumber-1.2.0/bin/../lib/cucumber/runtime/support_code.rb:171:in `load_file'
org/jruby/RubyArray.java:1613:in `each'
/var/lib/jenkins/.rvm/gems/jruby-1.7.9@cirrus-new/gems/cucumber-1.2.0/bin/../lib/cucumber/runtime/support_code.rb:83:in `load_files!'
/var/lib/jenkins/.rvm/gems/jruby-1.7.9@cirrus-new/gems/cucumber-1.2.0/bin/../lib/cucumber/runtime/support_code.rb:82:in `load_files!'
/var/lib/jenkins/.rvm/gems/jruby-1.7.9@cirrus-new/gems/cucumber-1.2.0/bin/../lib/cucumber/runtime.rb:175:in `load_step_definitions'
/var/lib/jenkins/.rvm/gems/jruby-1.7.9@cirrus-new/gems/cucumber-1.2.0/bin/../lib/cucumber/runtime.rb:40:in `run!'
/var/lib/jenkins/.rvm/gems/jruby-1.7.9@cirrus-new/gems/cucumber-1.2.0/bin/../lib/cucumber/cli/main.rb:43:in `execute!'
/var/lib/jenkins/.rvm/gems/jruby-1.7.9@cirrus-new/gems/cucumber-1.2.0/bin/../lib/cucumber/cli/main.rb:20:in `execute'
org/jruby/RubyKernel.java:1099:in `load'
/var/lib/jenkins/.rvm/gems/jruby-1.7.9@cirrus-new/gems/cucumber-1.2.0/bin/cucumber:14:in `(root)'
org/jruby/RubyKernel.java:1119:in `eval'
/var/lib/jenkins/.rvm/gems/jruby-1.7.9@cirrus-new/bin/jruby_executable_hooks:15:in `(root)'
````

**Comment by asofian** [comment]
I had a similar issue while running ruby-based selenium rake tasks on a CI server (Jenkins). I would run my rake tasks on Jenkins using the following command:

         xvfb-run --server-args="-screen 0, 1920x1280x24" rake example

What I failed to realize was that the headless gem itself opens and closes xvfb on its own; removing the gem completely from my code and running the same command proved successful. Simply omitting the 'xvfb-run' command and its arguments would likely have solved the issue as well - I chose to remove the gem because I was running tests on Jenkins and as well as on my local machine, on which I was not running headless-ly.
